charts/ccd-data-store-api/values.aat.template.yaml

```diff
@@ -1,5 +1,4 @@
 java:
-  replicas: 2
```

Comment:
shouldn't AAT be an exact replica of Prod?

Reply:
This config is staging pod only which is used by functional tests only.

Reply:
yes, aat.template.yaml is used only for staging deployment where tests are run. Its a bit redundant config which we will try to get rid of soon.

Reply:
👍 That'd be great @adusumillipraveen if you can remove it. Much less maintainability effort